### PR TITLE
add developer documentation on website

### PR DESCRIPTION
**Pull request by thperret**
move out website build commands from README to website
add commands to install compatible Sphinx version

### CODE REVIEW

File: doc/sphinx/source/installation_ubuntu1204.rst [hunk 1]
```diff
@@ -37,7 +37,14 @@ Install the following packages:
...
+    $ sudo apt-get install python-scikits-learn python-joblib python-pygraph python-pygraphviz python-PIL python-munkres python-paramiko
+
+This dependency is too old on the packages system manager::
+
+    $ pip install --user sphinx
+
+if you already installed sphinx with the packages manager, add the flag ``--upgrade`` to the previous command
+
 
 If you plan to use our specific viewer (pyhrf_viewer), run:
```
Comment (philouc) [inline doc/sphinx/source/installation_ubuntu1204.rst]: Should we remove this line??

File: doc/sphinx/source/installation_ubuntu1404.rst [hunk 2]
```diff
@@ -30,7 +30,14 @@ Install the following packages:
...
+    $ sudo apt-get install python-scikits-learn python-joblib python-pygraph python-pygraphviz python-PIL python-munkres python-paramiko
+
+This dependency is too old on the packages system manager::
+
+    $ pip install --user sphinx
+
+if you already installed sphinx with the packages manager, add the flag ``--upgrade`` to the previous command
+
 
 If you plan to use our specific viewer (pyhrf_viewer), run:
```
Comment (philouc) [inline doc/sphinx/source/installation_ubuntu1404.rst]: Should we remove this line? or instead give the right command line below??

**Comment by philouc** [comment]
Important step completed. Very useful to have the doc online for further developments.
Take care about my comments related to pyhrf_view

**Comment by thperret** [comment]
Thanks for your comments.
About the viewer, do we want to completely remove it?
Otherwise I'll fix the command line and add the proper links to documentation.

**Pull request merged**